ui/analyse/css/_action-menu.scss

```diff
@@ -38,30 +39,43 @@
   }
 
   &__tools {
-    flex: 0 0 auto;
-    display: flex;
-    flex-flow: row wrap;
-
-    > * {
-      @extend %flex-column;
+    display: grid;
+    gap: 0 6px;
+    grid-auto-rows: 1fr;
+    grid-template-columns: auto auto;
 
-      flex: 0 0 50%;
-      text-align: center;
-      padding: 0;
-      height: 75px;
-      justify-content: center;
+    @container (max-width: 280px) {
```

Comment:
that too https://developer.mozilla.org/en-US/docs/Web/CSS/@container#browser_compatibility